### DISCUSSION TITLE
Can this drag-and-drop multi-level menu management be built into v3 by default or optional?

### DISCUSSION THREAD

**Discussion by killjin** [filamentphp/filament#7017] [Ideas]
Can this drag-and-drop multi-level menu management be built into v3 by default or optional?
![multiLevelMenu](https://github.com/filamentphp/filament/assets/2804854/eb645633-e4da-4679-8b56-c89b63fd5f57)

**Comment by danharrin** [comment]
It's already in this plugin https://github.com/ryangjchandler/filament-navigation

**Reply by koti42** [reply]
This package is not working at the moment, in fact, this issue can be brought up again Navigation menus have a drag and drop structure
